index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Apple-page</title>
+    <link rel="stylesheet" href="styles/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    
+</head>
+<body>
+	
+<div id="content" >
+	<div id="content-2">
+	<nav id="nav-buttons">
+			<div id="global-nav">
+			<div id="gn-apple"><a class="nav-icons" href="#"><i class="fa fa-apple" style="font-size:24px; color: white"></i></a></div>
+			<div id="gn-store"><a class="nav-icons" href="#"><span>Store</span></a></div>
+			<div id="gn-mac"><a class="nav-icons" href="#"><span>Mac</span></a></div>
+			<div id="gn-ipod"><a class="nav-icons" href="#"><span>iPod</span></a></div>
+			<div id="gn-iphone"><a class="nav-icons" href="#"><span>iPhone</span></a></div>
+			<div id="gn-ipad"><a class="nav-icons" href="#"><span>iPad</span></a></div>
+			<div id="gn-itunes"><a class="nav-icons" href="#"><span>iTunes</span></a></div>
+			<div id="gn-support"><a class="nav-icons" href="#"><span>Support</span></a></div>
+			</div>
+			<div id="global-search">
+				<form id="search-form" action="#" method="post"><i class="fa fa-search" style="width:  15px; height: 14px; margin-left: 6px; margin-top: 3px;"></i>
```

Comment:
[REQUIRED] Remove inline styles Include all your styles in the external CSS file